### DISCUSSION TITLE
Unable to change Fieldclass fields (locked)

### DISCUSSION THREAD

**Discussion by Dennyyusuf** [mdelobelle/metadatamenu#680] [General]
Hello,

I am not sure whether there is a setting for this.
After setting up a fieldclass field I am not able to change this field later? it shows locked?

<img width="466" alt="image" src="https://github.com/mdelobelle/metadatamenu/assets/65077044/2bbf9fbe-6a63-41e9-b2bb-3a99ef483219">

**Comment by LynetteCullens** [comment]
Rename it to something else, create a new one with the original name, and then delete the old one.

**Reply by Dennyyusuf** [reply]
Hmm not sure but it does not work for me. Thanks btw for the advice

**Reply by LynetteCullens** [reply]
> Hmm not sure but it does not work for me. Thanks btw for the advice

It doesn't work for anyone. I think it's a feature. We can't change anything except the name and queries after creation.
